main/gradebook/gradebook_display_certificate.php

```diff
@@ -126,23 +126,28 @@ function confirmation() {
         $content = $form->returnForm();
         break;
     case 'export_all_certificates':
-        if (api_is_student_boss()) {
-            $userGroup = new UserGroup();
-            $userList = $userGroup->getGroupUsersByUser(api_get_user_id());
+        if ($allowCustomCertificate) {
+            $params = 'course_code='.api_get_course_id().'&session_id='.api_get_session_id().'&'.api_get_cidreq();
+            $url = api_get_path(WEB_PLUGIN_PATH).'customcertificate/src/print_certificate.php?export_all_in_one=1&'.$params;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line exceeds maximum limit of 120 characters; contains 124 characters](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5687817)